source/scripts/analytics.js

```diff
@@ -0,0 +1,91 @@
+// Run the init() function when the page has loaded
+window.addEventListener("DOMContentLoaded", init);
+
+// Starts the task program, all function calls trace back here
+function init() {
+	let tasks = JSON.parse(localStorage.getItem("tasks")) || [];
+	// Add each task to the <tbody> element
+	addTaskToDocument(tasks);
+	// Add the event listeners to the form elements
+}
+
+/**
+ * Takes in an array of tasks and for each task creates a
+ * new <to-do-task> element, adds the task data to that item
```

Comment:
I know this was copied over from the original function in add.js but this comment is extremely outdated as we no longer use a customized element (apologies on my part since Tianya and I never updated this when we created these functions). I also just realized this is now the same `addTaskToDocument()`  function as the one in add.js, so can we reuse the same one by importing the function to this analytics file?

Reply:
The two functions are not the same. For this same function in add.js, the difference must be less than 0 in order for the task to be added to the page. For this function in analytics.js, the difference must be more than 0 in order for the task to be added to the page. Basically the difference being less than 0 tells us that the task hasn't been finished yet so we add it to the table in the home page in add.js. Likewise the difference being more than 0 in analytics.js tells us that the task is finished so add it to the table in analytics page.

Reply:
soryr im blind did not see clearly but thanks for the explanation !